src/m365/context/commands/context-init.ts

```diff
@@ -0,0 +1,66 @@
+
+import * as fs from 'fs';
+import { Logger } from '../../../cli/Logger';
+import Command from '../../../Command';
+import { M365RcJson } from '../../base/M365RcJson';
+import commands from '../commands';
+
+class ContextInitCommand extends Command {
+  public get name(): string {
+    return commands.INIT;
+  }
+
+  public get description(): string {
+    return 'Retrieve context from Microsoft 365';
+  }
+
+  constructor() {
+    super();
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    await this.saveContextInfo({}, logger);
+  }
+
+  private saveContextInfo(contextInfo: any, logger: Logger): Promise<any> {
```

Comment:
Since we'll be needing this logic in other context commands, let's extract it to a baseclass, similarly to how we've done it with the `m365 app...` commands